opendevin/events/action/agent.py

```diff
@@ -105,6 +105,17 @@ def message(self) -> str:
         return "All done! What's next on the agenda?"
 
 
+@dataclass
+class AgentRejectAction(Action):
+    outputs: Dict = field(default_factory=dict)
```

Comment:
I don't think this should have an arbitrary dict of outputs, the way `finish` does. All we want is a `reason`

Reply:
Will create a follow-up to enforce the schema for micro-agents. We could discuss if we want to continue this approach there.